README.md

```diff
@@ -15,6 +15,10 @@ RepoSense is a contribution analysis tool for Git repositories. It is particular
   - [User Guide for the latest `master` (not yet released to users)](https://reposense.github.io/RepoSense)
 
 ---
+### Our Contributors :
+<a href="https://github.com/ckcherry23/RepoSense/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=ckcherry23/RepoSense" />
+</a>
```

Comment:
Hi @Kalyanimhala , I personally think that this is a nice addition. For the links, it should point to the main repo on RepoSense rather than a fork of RepoSense.

```suggestion
<a href="https://github.com/reposense/RepoSense/graphs/contributors">
  <img src="https://contrib.rocks/image?repo=reposense/RepoSense" />
</a>
```

Reply:
@Kalyanimhala please resolve this issue so that we can merge the PR